yesod-auth-hashdb 1.7.1.1 test failure

```
Preprocessing library for yesod-auth-hashdb-1.7.1.1..
    Building library for yesod-auth-hashdb-1.7.1.1..
    Preprocessing test suite 'integration' for yesod-auth-hashdb-1.7.1.1..
    Building test suite 'integration' for yesod-auth-hashdb-1.7.1.1..
    [1 of 4] Compiling TestSite [Yesod.Auth.Message changed]

    /var/stackage/work/unpack-dir/unpacked/yesod-auth-hashdb-1.7.1.1-579d3747ca2461d5b6312a3aa199306cb7ac0b574e039df29b4c382362e6310d/test/TestSite.hs:33:1: error:
        Generating Persistent entities now requires the following language extensions:

    DerivingStrategies
    StandaloneDeriving

    Please enable the extensions by copy/pasting these lines into the top of your file:

    {-# LANGUAGE DerivingStrategies #-}
    {-# LANGUAGE Standal
```

cc @paul-rouse

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -5306,7 +5306,6 @@ expected-test-failures:
     - yeshql-core # https://github.com/tdammers/yeshql/issues/6
     - yeshql-hdbc # https://github.com/tdammers/yeshql/issues/6
     - yesod-gitrev # https://github.com/DanBurton/yesod-gitrev/issues/5
-    - yesod-auth-hashdb # https://github.com/commercialhaskell/stackage/issues/5047
 
     # Recursive deps https://github.com/fpco/stackage/issues/1818
     - options

